docs/defi/flow-yield-vaults/defi-actions.md

```diff
@@ -0,0 +1,440 @@
+---
+title: DeFi Actions
```

Comment:
Perhaps we should rename this to FYV DeFi actions because we already have a Flow Actions section of the docs